### ISSUE TITLE
reWASD

### ISSUE THREAD

**Issue by Zaico89** [EugeneSunrise/reWASD#4]
hello when you put reWASD crack for windows 1

**Comment by EugeneSunrise** [comment]
Releases, checl readme brotha

**Comment by FrostyGITHUB** [comment]
Will be a crack for Windows or do I have to reset the trial every 3 day?

**Comment by EugeneSunrise** [comment]
> Will be a crack for Windows or do I have to reset the trial every 3 day?

I will do my best, but for now if you have x86 or x64 just reset the trial(you can also make a script to automate)

**Comment by yukishiro01** [comment]
I want x86/x64 version

**Comment by EugeneSunrise** [comment]
> Могу помочь как нибудь, чтобы ускорить релиз для x86 x64? 
> 
> И еще вопрос, работают ли фичи в этом кряке? Фичи: https://www.rewasd.com/advanced-controller-mapping

Можно помочь если есть опыт в .net реверсе, в кряке все фичи работают, даже мобильный контроллер

**Comment by SugaR-ddy** [comment]
Are you working with the x64/x84 ? How long will it take you to finish

**Comment by EugeneSunrise** [comment]
> Are you working with the x64/x84 ? How long will it take you to finish

In process, can't give an exact date...

**Comment by 3laar** [comment]
what is the current state of x64 version?
thank you for all your efforts

**Comment by EugeneSunrise** [comment]
> what is the current state of x64 version?
> thank you for all your efforts

At the stage of choosing code modification variations, versions for x64/x86 and arm are different and therefore require different approach, in case of x86/x64 everything is a bit more complicated + in addition I don't have a pc on x86/x64 now, it also adds time

**Comment by brunogallo** [comment]
Any news about x86/x64?

**Comment by EugeneSunrise** [comment]
> Извините что спрашиваю,но сброс триала не работает на материнских платах Асус,так как серийный номер платы сбрасывает при перезагрузке компьютера. Не знаете как это обойти?

https://zelenka.guru/threads/4722527/          
Read the thread, maybe it will help you

**Comment by EugeneSunrise** [comment]
> Any news about x86/x64?

In the process.

**Comment by EugeneSunrise** [comment]
DM ME IN TELEGRAM RIGHT NOW BEFORE I GO TO SLEEP, I NEED TO TEST A VERSION

**Comment by EugeneSunrise** [comment]
DM ME IN TELEGRAM RIGHT NOW BEFORE I GO TO SLEEP, I NEED TO TEST A VERSION

**Comment by 3laar** [comment]
> DM ME IN TELEGRAM RIGHT NOW BEFORE I GO TO SLEEP, I NEED TO TEST A VERSION
Where is your telegram? 
https://t.me/rafat3laa295

**Comment by HitmanAgentICA47** [comment]
https://t.me/HitmanICA

**Comment by EugeneSunrise** [comment]
Still not work, in progress...

**Comment by QHYSC** [comment]
refuel

**Comment by Emilia33** [comment]
anyone know any working trial reset for rewasd?

**Comment by HitmanAgentICA47** [comment]
> anyone know any working trial reset for rewasd?

Yes. He already wrote it on the main page. For specific motherboards (for example Asus), you have to dump your bios

**Comment by theroughguy** [comment]
> Yes. He already wrote it on the main page. For specific motherboards (for example Asus), you have to dump your bios
Hey man is this method still working i tried it but was facing some issues ho to get it done properly if you know the method can you guide me

**Comment by theroughguy** [comment]
> > anyone know any working trial reset for rewasd?
> 
> Yes. He already wrote it on the main page. For specific motherboards (for example Asus), you have to dump your bios

Hey man is this method still working i tried it but was facing some issues ho to get it done properly if you know the method can you guide me

**Comment by EugeneSunrise** [comment]
> > > anyone know any working trial reset for rewasd?
> > 
> > 
> > Yes. He already wrote it on the main page. For specific motherboards (for example Asus), you have to dump your bios
> 
> Hey man is this method still working i tried it but was facing some issues ho to get it done properly if you know the method can you guide me

The method works, I've tested it myself

**Comment by DDGAMEDD1** [comment]
while waitin for 64/84x version is there any alternative to use Keyboard as controller?

**Comment by gtxg16** [comment]
Yea method works but it takes SO LONG

**Comment by DDGAMEDD1** [comment]
waitin for 64x version still

**Comment by Zizoplays1** [comment]
> while waitin for 64/84x version is there any alternative to use Keyboard as controller?

There is Xoutput, works better 100 times more than X360ce, but not better than reWASD, I would say it's about the same level like it but features like PS/NS controls and button layouts, vibration for generic controllers, show on-screen inputs and real-time button presses are benefits that unfortunately Xoutput does not have

**Comment by 3laar** [comment]
> while waitin for 64/84x version is there any alternative to use Keyboard as controller?

You can use UCR it does the jop but its beta only

**Comment by 3laar** [comment]
> > waitin for 64x version still
> 
> I don't think he will ever release it honestly. It's been already 3 months and there is no news. He probably couldn't crack it.

TBH, Its hard to crack x64 on arm machine

**Comment by EugeneSunrise** [comment]
Guys, crack will be definitely, but you need to wait, very busy right now and too much work

**Comment by DDGAMEDD1** [comment]
> > while waitin for 64/84x version is there any alternative to use Keyboard as controller?
> 
> There is Xoutput, works better 100 times more than X360ce, but not better than reWASD, I would say it's about the same level like it but features like PS/NS controls and button layouts, vibration for generic controllers, show on-screen inputs and real-time button presses are benefits that unfortunately Xoutput does not have

I want to use keyboard as controller is it possible with xoutput?

**Comment by Zizoplays1** [comment]
Yes, I'm sure

**Comment by DDGAMEDD1** [comment]
Is there anything new about 64x?

**Comment by bjztbest** [comment]
Keep strong，  we are looking forward to it, we support you, thank you.

**Comment by GoodMan13iq** [comment]
I'm still waiting for 64\84x, any news? And is it worth the wait?

**Comment by u8Jj3** [comment]
Stop asking for ETA this will not bring reWASD faster.... Support @EugeneSunrise for his work, cracking software take time and asking for it in a github issue is not a thing to do

> I'm still waiting for 64\84x, any news? And is it worth the wait?

If you think this is not worth the wait you can buy the software ?

**Comment by Nabeeh-AlSawaf** [comment]
Thanks for the hard work, hoping u manage to crack it cheers!~ (i know issues aint meant for cheering as well but hey i wanna subscribe to the issue and make sure the developer knows he is appreciated no matter what !)
